judo-tatami-jsl-jsl2psm/src/main/epsilon/transformations/psm/modules/data/containment.etl

```diff
@@ -12,3 +12,62 @@ rule CreateContainmentFromField
         s.eContainer.getPSMEquivalent().relations.add(t);
         log.debug("Created Containment: " + t.name);
     }
+
+//////////////////////////////////////////////////////////////////
+// Entity TransferField Reads
+//////////////////////////////////////////////////////////////////   
+@greedy
+rule CreateReadsDataExpressionTypeForTransferRelationDeclaration
```

Comment:
We are mapping to `JUDOPSM!ReferenceExpressionType`, so not sure why are we calling it "DataExpressionType"